youtube_dl/extractor/motherless.py

```diff
@@ -114,3 +116,72 @@ def _real_extract(self, url):
             'age_limit': age_limit,
             'url': video_url,
         }
+
+
+class MotherlessGroupIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?motherless\.com/gv?/(?P<id>[a-z0-9_]+)/?$'
```

Comment:
`/?$` senseless. Does not match http://motherless.com/gv/sex_must_be_funny?foo=bar.

Reply:
It may not match `http://motherless.com/g/cosplay/633979F` and I'm a bit at a loss on how to accomplish that.